web/views/group_view.ex

```diff
@@ -25,4 +25,14 @@ defmodule Pairmotron.GroupView do
     conn.assigns.current_user.group_membership_requests
     |> Enum.find(&(&1.group_id == group.id and not &1.initiated_by_user))
   end
+
+  def truncate(nil, _), do: nil
+  def truncate(string, len) do
+    cond do
+      String.length(string) > len ->
+        String.slice(string, 0, len) <> "..."
+      True ->
+        string
+    end
+  end
```

Comment:
Good call on this. I've added an overflow: scroll property to those boxes. It looks like this, with a scroll bar on the bottom if you hover over it:

<img width="790" alt="screen shot 2017-03-02 at 2 35 51 pm" src="https://cloud.githubusercontent.com/assets/4420576/23523726/142592aa-ff56-11e6-8897-ec7830ec9f25.png">

<img width="778" alt="screen shot 2017-03-02 at 2 36 06 pm" src="https://cloud.githubusercontent.com/assets/4420576/23523727/14269ec0-ff56-11e6-9148-985511ca47e4.png">

Normally, I think truncating to 110 will catch most cases of this. I'm up for completely different solutions for this, but we should definitely limit this in one way or another.